README.md

```diff
@@ -10,6 +10,7 @@ This gem provides (via a Rails engine):
 * Admin design patterns available from `/style-guide` (when routes are mounted)
 * [CSS helpers and SASS variables](CSS.md) for the admin theme
 * GOV.UK user friendly date formats
+* Google Analytics tracking code (Universal Analytics), using the "GOV.UK apps" profile ID `UA-26179049-6`
```

Comment:
Not sure about the hard-coding of the profile ID here - maybe just deep-link to `/govuk_admin_template.html.erb`?